ledger-bridge.js

```diff
@@ -4,10 +4,19 @@ require('buffer')
 import TransportU2F from '@ledgerhq/hw-transport-u2f'
 import LedgerEth from '@ledgerhq/hw-app-eth'
 import { byContractAddress } from '@ledgerhq/hw-app-eth/erc20'
+import WebSocketTransport from '@ledgerhq/hw-transport-http/lib/WebSocketTransport'
```

Comment:
Would this work instead? I'd be nice to not rely on the exact file path.

```suggestion
import { WebSocketTransport } from '@ledgerhq/hw-transport-http'
```

Reply:
I can give this a try; I stuck mostly to their pattern from the demo.